### PR TITLE
Add option to show forecast icon

### PR DESCRIPTION
**Pull request by smalldave**
This is text mode goddammit!

Defaults to true so behaviour is as before.

### CODE REVIEW

**Comment by smalldave** [comment]
Hi,
Realise this is probably pretty low priority.
Any thoughts on this?
Cheers.

File: main.go [hunk 1]
```diff
@@ -47,6 +48,7 @@ func init() {
 	flag.IntVar(&days, "days", 0, "No. of days to get forecast")
 	flag.IntVar(&days, "d", 0, "No. of days to get forecast (shorthand)")
 	flag.BoolVar(&ignoreAlerts, "ignore-alerts", false, "Ignore alerts in weather output")
+	flag.BoolVar(&showIcon, "show-icon", true, "Show an icon representing the weather")
```
Comment (thaJeztah) [inline main.go]: How about `hide-icon`? For boolean flags it feels more natural to make the "value" optional. i.e., `--hide-icon` versus `--show-icon=false`

Comment (jessfraz) [inline main.go]: ooo i like that, sorry i completely missed this

**Comment by jessfraz** [comment]
thanks!

**Pull request merged**